runtime/JavaScript/src/antlr4/Lexer.js

```diff
@@ -204,6 +219,9 @@ Lexer.prototype.skip = function() {
 };
 
 Lexer.prototype.more = function() {
+	if (PORT_DEBUG) {
+		console.log("more")
+	}
```

Comment:
All of the JavaScript changes will be removed when submitting the PR. This is the best way I've found to debug the Go implementation.